linera-core/src/client/mod.rs

```diff
@@ -2172,7 +2124,7 @@ where
         let committee = self.local_committee().await?;
         let max_size = committee.policy().maximum_block_proposal_size;
         block.check_proposal_size(max_size)?;
```

Comment:
Do we check that `blobs` for this particular `block` are also within the size limits?

Reply:
Even though they're threaded separately, they are part of the same block proposal.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3242/commits/f097f389c5da0c91605bdad1e68ccb47a319de41.